.github/workflows/pr.yml

```diff
@@ -0,0 +1,58 @@
+name: "PR Tests"
+permissions: read-all
+
+# Trigger for PR an merge to develop branch
+on:
+  push:
+    branches: develop
+  pull_request:
+  workflow_dispatch:
+
+env:
+  CTEST_OUTPUT_ON_FAILURE: 1
+  LAPACK_VERSION: 3.12.0
+  PARALLEL: -j 2
```

Comment:
What sort of system will this run on? If we can increase the number of parallel jobs, that should significantly decrease the build time of tests.

Reply:
github provided runners have 1 core, 2 threads. You can have up to 20 concurrent jobs. There is a way to pay for runners with more cores. I don't think it would be expensive. Someone is looking into getting a dedicated node in devcloud that we could use. Then 1 runner could use 120 cores and 4 GPU's. This would solve most of our problems so I am hoping we could live with slow performance until that happens.

Reply:
> What sort of system will this run on? If we can increase the number of parallel jobs, that should significantly decrease the build time of tests.

The new github org: `onemkl-project` has runners for a 24 core system. It reduces the total time to 30 minutes. Still working on PVC and more cores.
https://github.com/onemkl-project/ci-test/actions/runs/9140509510